### DISCUSSION TITLE
Running a project without clone.

### DISCUSSION THREAD

**Discussion by ghost** [pyscript/pyscript#1906] [Q&A]
Is it acceptable if i run a project https://pyscript-dev.com/@examples, https://pyscript.com/@examples without clone just copying index.html and main.py?
poc:
https://[nasrin1748.github.io/pyscript_dataframe/](https://nasrin1748.github.io/pyscript_dataframe/)

**Comment by ghost** [comment]
Can this be blocked as well https://nasrin1748.github.io/pyscript-test/

**Reply by WebReflection** [reply]
what do you mean by *blocked* ? 🤔

**Reply by ghost** [reply]
https://_12.pyscriptapps.com/little-math/latest/ shows like blocked on moblie .Shows the following image.
<img width="77" alt="Screenshot 2024-01-03 045055" src="https://github.com/pyscript/pyscript/assets/79505893/e9b67580-57d9-4cdc-ab4f-79a0068cc6b6"> 
Just gave another challenge. https://nasrin1748.github.io/pyscript-test/
